src/components/map/airports/MapAirportRunwaySelector.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div class="runways-container">
+        <common-notification
+            cookie-name="vatglasses-runways"
+            type="info"
+        >
+            In this block you can select active runways<br> for VatGlasses sectors
```

Comment:
I would call it "active runway config", to make it more clear that for example in EDDB 06  and 24 means 06L/R and 24L/R and 06 and 24 does not actually mean the runways.